apps/screloc/main.cpp

```diff
@@ -571,6 +583,7 @@ class Reloc : public Client::Application {
 		ObjectLog                 *_inputOrgs;
 		ObjectLog                 *_outputOrgs;
 		bool                       _useWeight;
+		bool                       _storeSourceOriginID;
```

Comment:
It needs a default value, either provided here as:

```
bool   _storeSourceOriginID{false};
```

or initialized in the constructor similar to `_useWeight`.

Reply:
thanks!